.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Would it be possible to make this setting using EditorConfig?

Reply:
It needs an [additional extension](https://marketplace.visualstudio.com/items?itemName=EditorConfig.EditorConfig) in VS Code but i'm ok with that and have updated the PR